### DISCUSSION TITLE
unable to install via brew

### DISCUSSION THREAD

**Discussion by divyar176** [Homebrew/discussions#2168] [Getting started]
Hey guys i have installed brew but unfortunately i am not able to install node and mongodb through brew. any leads?
<img width="560" alt="Screen Shot 2021-09-20 at 4 35 29 PM" src="https://user-images.githubusercontent.com/62485459/134072055-388dcb0e-01d3-460e-aea5-d018aee10e2f.png">

**Comment by SMillerDev** [comment]
Did you actually do the things it says under "next steps"?

**Reply by divyar176** [reply]
yes i did it now gives me this error
<img width="540" alt="Screen Shot 2021-09-20 at 6 20 26 PM" src="https://user-images.githubusercontent.com/62485459/134083628-af6af7af-11e9-4682-8975-9f10cb4b9d6f.png">

**Reply by SMillerDev** [reply]
So I'll assume that `brew install nodejs` works now. The reason it can't find MongoDB is that MongoDB isn't actually open source and the homebrew project does not have it by default.
The MongoDB project has install documentation here: 
https://docs.mongodb.com/manual/tutorial/install-mongodb-on-os-x/
